ui/package.json

```diff
@@ -8,8 +8,9 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "format": "prettier --write .",
```

Comment:
Hmm, sorry for the nudging, but also this commit does now more than "Add a script for formatting with Prettier":

1. It does also call `prettier` explicitly instead of using `--format`,
2. and reformats the code in `api/queries`.

IMO 1. should be done is a separate commit and 2. should be squashed to the next commit.